commands/channel_e2e_test.go

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package commands
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/mattermost/mattermost-server/v5/model"
+
+	"github.com/mattermost/mmctl/client"
+	"github.com/mattermost/mmctl/printer"
+)
+
+func (s *MmctlE2ETestSuite) TestCreateChannelCmd() {
+	s.SetupTestHelper().InitBasic()
+
+	s.RunForAllClients("create channel successfully", func(c client.Client) {
+		printer.Clean()
+		printer.SetFormat(printer.FormatPlain)
+
+		cmd := &cobra.Command{}
+		channelName := model.NewRandomString(10)
+		teamName := s.th.BasicTeam.Name
+		channelDisplayName := "channelDisplayName"
+		cmd.Flags().String("name", channelName, "channel name")
+		cmd.Flags().String("team", teamName, "team name")
+		cmd.Flags().String("display_name", channelDisplayName, "display name")
+
+		err := createChannelCmdF(c, cmd, []string{})
+		s.Require().Nil(err)
+		s.Require().Len(printer.GetErrorLines(), 0)
+		s.Require().Len(printer.GetLines(), 1)
+		s.Require().Equal("New channel "+channelName+" successfully created", printer.GetLines()[0])
+	})
```

Comment:
After successfully creating the channel, we should check that it was as well added to the database using `s.th.App` to fetch the channel and asserting that the name, display name and team are the same:

```go
newChannel, err := s.th.App.GetChannelByName(channelName, s.th.BasicTeam.Id, false)
...
```

Reply:
Added those :slightly_smiling_face: